projects/laji/src/app/shared/navbar/notifications/notifications.component.ts

```diff
@@ -20,10 +17,10 @@ import { DialogService } from '../../service/dialog.service';
 export class NotificationsComponent implements OnInit, OnDestroy {
   unsubscribe$ = new Subject<void>();
 
-  notificationSource: NotificationDataSource;
+  notificationSource!: NotificationDataSource;
 
-  @ViewChild(CdkVirtualScrollViewport, {static: true}) virtualScroll: CdkVirtualScrollViewport;
-  loading: boolean;
+  @ViewChild(CdkVirtualScrollViewport, {static: true}) virtualScroll!: CdkVirtualScrollViewport;
+  loading!: boolean;
```

Comment:
I think using `!` should be avoided when it's possible (with the exception of static `ViewChild`s or required `Input`s etc). In the other places it would probably require more refactoring to get rid of it but at least here it would be easy to convert it to `loading = false`

Reply:
I set it as `loading = true`, which makes more sense. It's loading from the beginning, until it's loaded. UI wise this has no difference, seems like the whole component isn't displayed before the facade's loading observable sends a value.